logging/thirdparty/src/test/java/io/quarkus/ts/logging/jboss/SyslogIT.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.ts.logging.jboss;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.QuarkusApplication;
+
+@QuarkusScenario
+public class SyslogIT {
+
+    @Container(image = "balabit/syslog-ng", port = 514, expectedLog = "syslog-ng")
+    static RestService syslog = new RestService()
+            .withProperty("_ignored", "resource_with_destination::/etc/syslog-ng/|syslog-ng.conf");
+    /*
+     * for manual testing: podman run -p 8514:514 -v $(pwd)/syslog-ng.conf:/etc/syslog-ng/syslog-ng.conf:z --rm -it
```

Comment:
this is useful, you can keep it as is

personally I wonder if it should be above `@Container` or in the module README, but it's not important